_posts/2022-11-07-changelog-182.markdown

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #182"
+date: 2022-11-07 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#9650] Increase the minimal supported version of Rust to 1.56
+
+* Procedural macro support improvements:
+  * [#9628] Enable the expansion of [function-like procedural macros] in the [nightly builds].
+    This feature can be controlled via `org.rust.macros.proc.function-like` option in `Experimental Features` dialog.
+    For more information, check the [implementation PR](https://github.com/intellij-rust/intellij-rust/pull/6564) and the corresponding [tracking issue](https://github.com/intellij-rust/intellij-rust/issues/6908).
+
+  * [#9526] Insert brackets of the proper form (`()`, `[]` or `{}`) when completing procedural macro calls
```

Comment:
Maybe add a short explanation that a particular brackets type depends on documentation